assert/assertions.go

```diff
@@ -19,6 +19,7 @@ import (
 
 	"github.com/davecgh/go-spew/spew"
 	"github.com/pmezard/go-difflib/difflib"
+	"github.com/stretchr/testify/internal/check"
```

Comment:
Move the import in a separate block below to separate internal dependencies and external ones.